web/src/people.js

```diff
@@ -926,6 +926,23 @@ export function get_bot_ids() {
 
     return bot_ids;
 }
+export function verify_non_active_human_id(user_id) {
+    if (non_active_user_dict.has(user_id)) {
+        return true;
+    }
+
+    return false;
+}
+
+export function get_deactivated_users_name(user_id) {
```

Comment:
The parameter should be called `user_ids`, should it appears to expect a list? 

Also the function name seems wrong, it should be `get_names_for_deactivated_users` or something like that, to hint the fact it expects a list.

Reply:
@timabbott Thanks for the suggestion, I have changed the name of the parameter and function, could you please review the PR, thanks :)